### PR TITLE
Fix streams

### PR DESCRIPTION
**Pull request by mprudnik**
- [x] tests and linter show no problems (`npm t`)
- [ ] tests are added/updated for bug fixes and new features
- [x] code is properly formatted (`npm run fmt`)
- [ ] description of changes is added in CHANGELOG.md
- [ ] update .d.ts typings

Done:
- `server` changes were discussed previously
- `client` used server-side receives binary packets as instances of `Buffer`, not a `Blob`.
- `MetaReadable` instantiation - just fixing how args are passed https://github.com/metarhia/metacom/blob/master/lib/streams.js#L28

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ class Metacom extends EventEmitter {
           if (stream) {
             console.error(new Error(`Stream ${name} is already initialized`));
           } else {
-            const stream = new MetaReadable({ id, name, size });
+            const stream = new MetaReadable(id, name, size);
```
Comment (tshemsedinov) [inline lib/client.js:121]: See MetaReadable constructor signature: https://github.com/metarhia/metacom/blob/86504033966fa73b3cb6733456c8d5ba6ced45af/lib/streams.js#L28-L29

Comment (tshemsedinov) [reply lib/client.js:121]: In client as well

**Review by tshemsedinov — COMMENTED** [review]
Going to land this

File: lib/client.js (hunk 1 above)
Comment (mprudnik) [reply lib/client.js:121]: @tshemsedinov what exactly you want me to see? we had invalid instantiation and I've fixed it

Comment (mprudnik) [reply lib/client.js:121]: @tshemsedinov MetaReadable accepts id, name and size as positional args. Before my update it was initialized with object as a first arg.

Comment (tshemsedinov) [reply lib/client.js:121]: Parameters in MetaReadable constructor are separate, not grouped into object, or not "named"

Comment (tshemsedinov) [reply lib/client.js:121]: Oh, sorry, my fault, yes, landing both commits

**Comment by tshemsedinov** [comment]
Landed in https://github.com/metarhia/metacom/commit/6ea9c227c2a56b321c2a30773babfb74498a1335 and https://github.com/metarhia/metacom/commit/94e0d248adf6a21b8f4e7c848d9dec95a5fd17c9

**Pull request closed without merging**